packages/react-native-codegen/src/parsers/typescript/components/componentsUtils.js

```diff
@@ -111,7 +111,11 @@ function detectArrayType<T>(
   typeAnnotation: $FlowFixMe | ASTNode,
   defaultValue: $FlowFixMe | void,
   types: TypeDeclarationMap,
-  buildSchema: (property: PropAST, types: TypeDeclarationMap) => ?NamedShape<T>,
+  buildSchema: (
+    property: PropAST,
+    types: TypeDeclarationMap,
+    language: $FlowFixMe,
```

Comment:
language is supposed to be a _string_ I guess :) same in the below cases as well...
```suggestion
    language: string,
```